configure.ac

```diff
@@ -191,6 +191,16 @@ AS_IF([test "x$with_docs_only" == xyes],
 
 
      #
+     # Enable multithreading support
+     #
+     AC_ARG_ENABLE([ucp-threads],
+                   AS_HELP_STRING([--enable-ucp-threads],
```

Comment:
I think we should have one global setting (e.g --enable-mt) and not per ucp/uct. @shamisp what do you think?